docs/documentation/new.md

```diff
@@ -119,15 +122,25 @@ Default applications are created in a directory of the same name, with an initia
   <p>
     The name of the source directory.
   </p>
+  <p>
+    You can change the value in <em>.angular-cli.json</em> (<code>apps[0].root</code>).
+  </p>
 </details>
 
 <details>
   <summary>style</summary>
   <p>
     <code>--style</code> <em>default value: css</em>
   </p>
+  <div>
+    The style file default extension. Possible values:
+    <ul>
+      <li>css</li>
+      <li>scss</li>
```

Comment:
@filipesilva I've updated the PR commit with less and stylus as options. I've stated `styl` for `stylus`, but I've written `styl (stylus)` to make it clear that it's clear.